content/docs/migration_1.0/Concepts.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Concepts"
+weight : 6100
+menu:
+  docs:
+    parent: migration_1.0
+---
+
+The Zenoh team have been hard at work preparing an official version 1.0.0 of Zenoh!   
+This major release comes with several API changes, quality of life improvements and developer conveniences.
+
+We now have a more stable API and intend to keep backward compatibility in the future Zenoh revisions.  
+This guide is here to ease the transition to Zenoh 1.0.0 for our users!
+
+## Value is gone, long live ZBytes 
+We have replaced `Value` with `ZBytes` and `Encoding`.  
+`Zbytes` is the type core to data representation in Zenoh, all API's have be reworked to accept `ZBytes` or something that can be converted into a `ZBytes`.  
+We have added a number of conversion implementations for language primitives as well as methods to seamlessly allow user defined structs to be serialized into `ZBytes`.  
+`Sample`'s payloads are now `ZBytes`, `Publishers`, `Queryables` and `Subscribers` now expect `ZBytes` for all their interfaces.
+
+Each Language bindings will have their own specifics of Serializing and Deserializing, but for the most part it will involve implementing a serialize / deserialize function for your datatype or make use of auto-generated conversations for composite types.
+
+## Encoding
+
+`Encoding` has been reworked. 
+Zenoh does not impose any encoding requirement on the user, nor does it operate on it.  
+It can be thought of as optional metadata, carried over by Zenoh in such a way that the end user’s application may perform different operations based on encoding.  
+We have expanded our list of predefined encoding types from Zenoh 0.11.0 to include variants for numerous IANA standard encodings, including but not limited to  `video/x` , `application/x`, `text/x`, `image/x` and `audio/x` encoding families, as well as an encoding family specific to Zenoh defined by the prefix `zenoh/x` .   
+Users can also define their own encoding scheme that does not need to be based on the predefined IANA variants. Upon declaring a specific encoding, the users encoding scheme will be prefixed with `zenoh/bytes` for distinction.
+
+
+## Attachment
+
+We have made attachment more flexible across API’s, essentially accepting anything that can be converted to a `ZBytes` as an optional extra to `put` , `delete` , on `Query`'s and Query `reply` s.  
+We also allow for composite types to be converted into `ZBytes`, meaning that using the Attachment API as metadata is more easy than ever.
+
+## Query & Queryable
+
+`Query` and `Queryable` have been slightly reworked.  
+For the API replying to a `Query` from a `Queryable` declared on a session: 
+The `reply` function has been split into 3 separate functions variants depending on the type of reply the user wants to send.  
```

Comment:
`reply` now behaves similar to `put` and `del` improving ergonomics compared to before.

Reply:
added